downstream/modules/platform/proc-attaching-subscriptions.adoc

```diff
@@ -0,0 +1,76 @@
+// emurtoug removed this assembly from the Planning guide to avoid duplication of subscription content added to Access management and authentication
+
+[id="proc-attaching-subscriptions"]
+
+= Attaching your {PlatformName} subscription
+
+[role="_abstract"]
+You *must* have valid subscriptions attached on all nodes before installing {PlatformName}. Attaching your {PlatformNameShort} subscription provides access to subscription-only resources necessary to proceed with the installation.
+
+NOTE: Attaching a subscription is unnecessary if you have enabled Simple Content Access Mode on your Red Hat account. Once enabled, you will need to register your systems to either Red Hat Subscription Management (RHSM) or Satellite before installing the {PlatformNameShort}. For more information, see link:https://access.redhat.com/articles/simple-content-access[Simple Content Access].
```

Comment:
The  Note isn't formatted correctly. Unless you intended it that way?

Reply:
Fixed.